Sources/Sentry/SentryUIViewControllerPerformanceTracker.m

```diff
@@ -109,6 +120,55 @@ - (void)createTransaction:(UIViewController *)controller
     }
 }
 
+- (void)createTimeToDisplay:(UIViewController *)controller
+{
+    SentrySpan *vcSpan = [self viewControllerPerformanceSpan:controller];
+
+    if (![vcSpan isKindOfClass:[SentryTracer self]]) {
+        // Since TTID and TTFD are meant to the whole screen
+        // we will not track child view controllers
```

Comment:
💯 for this vital comment.

Reply:
The comment is informative, but I have no idea how it is related to the class type of the span. We should not be wrapping up vital logic like that in the type system like this.